.github/workflows/rust.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           - stable
           - 1.64.0
         indexmap:
-          - 1.9.3
+          - 2.0.0
```

Comment:
I don't know if we need to try 2.0.0 in particular? This was added to test a 1.x.x version and a "recent" 2.x.x version. So probably this whole strategy variable can be deleted